KtSPR--L5-T1--авторизация

DenisGubernatorov opened the pull request:
Реализация пользователя через решение математического выражения

DenisGubernatorov committed:
KtSPR--L5-T1--авторизация
Реализация пользователя через решение математического выражения

ievetrovdev commented on src/main/kotlin/lesson5/lesson5_task1.kt:
```diff
@@ -0,0 +1,14 @@
+const val CORRECT_ANSWER = 12
+
+fun main() {
+    print("Для авторизации введите результат выражения $FIRST_AUTH_NUMBER + $SECOND_AUTH_NUMBER = ")
```
Правильно что числа в отдельных переменных, но делать их константами неправильно. Потенциально эти переменные должны быть разными при каждой авторизации

ievetrovdev commented on src/main/kotlin/lesson5/lesson5_task1.kt:
```diff
@@ -0,0 +1,14 @@
+fun main() {
+    print("Для авторизации введите результат выражения $FIRST_AUTH_NUMBER + $SECOND_AUTH_NUMBER = ")
+    val result = readln().toInt()
+    when (result) {
```
это не ошибка, но обычно when используют когда вариантов больше 2

DenisGubernatorov committed:
KtSPR--L5-T1--авторизация
Доработки согласно замечаниям
1) Слагаемые перенесены из констант в простые переменные
2) Замена when на if
